src/Symfony/Component/Ldap/Adapter/AdapterInterface.php

```diff
@@ -44,11 +44,11 @@ public function getEntryManager();
     /**
      * Escape a string for use in an LDAP filter or DN.
      *
-     * @param string $subject
-     * @param string $ignore
-     * @param int    $flags
+     * @param string      $subject
+     * @param string|null $ignore
+     * @param int         $flags
```

Comment:
The `@param` annotations are redundant now and may be removed.

Reply:
fixed :)